src/openrct2-ui/windows/Staff.cpp

```diff
@@ -119,1240 +121,1169 @@ static rct_widget* window_staff_page_widgets[] = {
+                window_staff_options_widgets[WIDX_CHECKBOX_3].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_CHECKBOX_4].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_COSTUME_BOX].type = WindowWidgetType::DropdownMenu;
+                window_staff_options_widgets[WIDX_COSTUME_BTN].type = WindowWidgetType::Button;
+                window_staff_options_widgets[WIDX_COSTUME_BOX].text = StaffCostumeNames[EnumValue(_staff->SpriteType) - 4];
+                break;
+            case StaffType::Handyman:
+                window_staff_options_widgets[WIDX_CHECKBOX_1].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_1].text = STR_STAFF_OPTION_SWEEP_FOOTPATHS;
+                window_staff_options_widgets[WIDX_CHECKBOX_2].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_2].text = STR_STAFF_OPTION_WATER_GARDENS;
+                window_staff_options_widgets[WIDX_CHECKBOX_3].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_3].text = STR_STAFF_OPTION_EMPTY_LITTER;
+                window_staff_options_widgets[WIDX_CHECKBOX_4].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_4].text = STR_STAFF_OPTION_MOW_GRASS;
+                window_staff_options_widgets[WIDX_COSTUME_BOX].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_COSTUME_BTN].type = WindowWidgetType::Empty;
+                pressed_widgets &= ~(
+                    (1ULL << WIDX_CHECKBOX_1) | (1ULL << WIDX_CHECKBOX_2) | (1ULL << WIDX_CHECKBOX_3)
+                    | (1ULL << WIDX_CHECKBOX_4));
+                pressed_widgets |= static_cast<uint64_t>(_staff->StaffOrders) << static_cast<uint64_t>(WIDX_CHECKBOX_1);
+                break;
+            case StaffType::Mechanic:
+                window_staff_options_widgets[WIDX_CHECKBOX_1].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_1].text = STR_INSPECT_RIDES;
+                window_staff_options_widgets[WIDX_CHECKBOX_2].type = WindowWidgetType::Checkbox;
+                window_staff_options_widgets[WIDX_CHECKBOX_2].text = STR_FIX_RIDES;
+                window_staff_options_widgets[WIDX_CHECKBOX_3].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_CHECKBOX_4].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_COSTUME_BOX].type = WindowWidgetType::Empty;
+                window_staff_options_widgets[WIDX_COSTUME_BTN].type = WindowWidgetType::Empty;
+                pressed_widgets &= ~((1ULL << WIDX_CHECKBOX_1) | (1ULL << WIDX_CHECKBOX_2));
+                pressed_widgets |= static_cast<uint64_t>(_staff->StaffOrders) << static_cast<uint64_t>(WIDX_CHECKBOX_1);
```

Comment:
Use `SetPressedWidget`

Reply:
I've fixed it for the `pressed_widgets &= ~((1ULL << WIDX_CHECKBOX_1) | (1ULL << WIDX_CHECKBOX_2));` but I have no idea how to do it for `pressed_widgets |= static_cast<uint64_t>(_staff->StaffOrders) << static_cast<uint64_t>(WIDX_CHECKBOX_1);`. 

I figure it would be better to use `SetCheckboxValue()` but since I'm quite new to bitwise operations I have no idea how to tell if the chechbox should be enabled or not